src/mixins/filing-template-mixin.ts

```diff
@@ -999,12 +999,14 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
    */
   private prepareShareClasses () : ShareClassIF[] {
     // filter out removed classes and delete "action" property
-    const shareClasses = this.getShareClasses.filter(x => x.action !== ActionTypes.REMOVED)
+    const shareClasses = this.getShareClasses
+      .filter(x => x.action?.toUpperCase() !== ActionTypes.REMOVED)
```

Comment:
`action` is set by a shared component. Here its value was `removed`, but the ActionTypes value is `REMOVED'.

This is a a quick fix. A longer fix would update the shared component and then find which projects use it and update those accordingly... future.

Reply:
I created /bcgov/entity#13170 to fix the greater issue around ActionTypes.